### ISSUE TITLE
Item list is not refreshed after adding a new item

### ISSUE THREAD

**Issue by wiegerjelsma** [verbb/navigation#93]
### Description
The list with navigation items in the left pane does not refresh after adding an item (entries, categories, assets as well as custom urls). The item is saved though. When refreshing the page the new item appears in the list.

### Steps to reproduce
1. Create or open a navigation.
2. Add an item to navigation.
3. Nothing changes.
4. Refreshing the page manually shows the new item in the navigation.

### Additional info

- Plugin version: 1.1.12
- Craft version: 3.1.29

**Comment by engram-design** [comment]
I wonder if there are any JS errors - could you check your web console to see if there are? Attached a quick video showing to working on my end.

https://www.dropbox.com/s/j6lmvqb6z3ub92i/navigation.mov?dl=0

**Comment by wiegerjelsma** [comment]
I guess it has to do with an incompatibility with another plugin. Maybe Commerce. I can't reproduce the error. I was just testing out some plugins and Commerce, so I don't know which one was causing the error. I'll let you know if happens again!

**Comment by wiegerjelsma** [comment]
I bumped into the issue again today. I've selected a parent and click the '+ Add an entry' button. After selecting the entry nothing happens. I've attached the error which appears in the console.
![Schermafbeelding 2019-08-07 om 10 13 56](https://user-images.githubusercontent.com/25075044/62766168-16155880-ba92-11e9-9472-83ecea4db9b1.png)
![Schermafbeelding 2019-08-09 om 10 39 03](https://user-images.githubusercontent.com/25075044/62766170-16adef00-ba92-11e9-9620-69b60a51afb7.png)
